apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.ts

```diff
@@ -146,7 +146,7 @@ export class AdvanceReceiptAdjustmentComponent implements OnInit {
      */
     public onCancel(): void {
         if (this.adjustPayment && this.adjustPayment.totalAdjustedAmount && this.adjustPayment.grandTotal && this.adjustPayment.totalAdjustedAmount - this.adjustPayment.grandTotal > 0) {
-            this.toaster.warningToast('The adjusted amount of the linked invoice\'s is more than this receipt');
+            this.toaster.warningToast('The adjusted amount of the linked invoice\'s is more than this receipt due amount');
```

Comment:
This same message is getting used multiple times in this file. Create a variable for this and use it .

Reply:
Both message is different please check

Reply:
Both is different message please check